sherpa/plot/__init__.py

```diff
@@ -32,7 +32,7 @@
 When this module is first imported, Sherpa tries to import the
 backends installed with Sherpa in the order listed in the
 ``options.plot_pkg`` setting from the ``sherpa.rc`` startup file.
-The first module that imports successfully is set as the active 
+The first module that imports successfully is set as the active
```

Comment:
Seems like an unrelated change. It's OK, I can rebase and deal with the conflict in #1382since you already wrote this... but it would be easier without it ;-)

Reply:
I think I had made a change in this file and then backed that change out, but the "make the file not have trailing spaces" logic made this change. I'll remove it.